Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
@omeshp IMHO best solution would be the possibility to define the **branch** used in autotrigger scenario at the time of linking. Until then I think always returning the default branch for LatestBuild is the best option, even though it's a limitation it at least leads to a predictable result.